src/Configuration.php

```diff
@@ -257,7 +257,13 @@ protected function createDevelopmentTag(string $path): string
     {
         $url = Str::of($this->config('dev_server.url'))->finish('/')->append($path);
 
-        if (Str::endsWith($path, '.css')) {
+        if (Str::endsWith($path, '.css')
+            || Str::endsWith($path, '.scss')
+            || Str::endsWith($path, '.sass')
+            || Str::endsWith($path, '.less')
+            || Str::endsWith($path, '.styl')
+            || Str::endsWith($path, '.stylus')
+        ) {
```

Comment:
```suggestion
        if (Str::endsWith($path, ['.css', '.scss', '.sass', '.less', '.styl', '.stylus']) {
```

Reply:
Can you describe exactly the scenario that happens when this check only looks for `.css`?

A new test would be ideal.